app/domain/rotation/rotators/gogs/user_account.rb

```diff
@@ -0,0 +1,72 @@
+return if defined? Rotation::Rotators::Gogs::UserAccount
+
+require 'rest-client'
+require 'securerandom'
+
+module Rotation
+  module Rotators
+    module Gogs
+      class UserAccount
+        def initialize()
+        end
+
+        def rotate(facade)
+          variable_id = facade.rotated_variable.resource_id
+          user_name = facade.annotations['gogs/user_name']
+          base_url = facade.annotations['gogs/url']
+          old_password = facade.current_values([variable_id])[variable_id]
+          new_password = new_password(base_url, user_name, old_password)
+          facade.update_variables({variable_id => new_password})
```

Comment:
My one major comment: You'll want to do something similar to this:

https://github.com/cyberark/conjur/blob/master/app/domain/rotation/rotators/postgresql/password.rb#L38

to ensure that the two updates (to conjur, and to gogs) occur as a unit.  Theoretically the way it currently is gogs could update and the conjur `update_variables` could fail and you'd be out of sync.